packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -44,6 +47,14 @@ contract MetaTransactionWallet is
     bytes returnData
   );
 
+  // onlyGuardian functions can only be called when the guardian is not the zero address and
+  // the caller is the guardian.
+  modifier onlyGuardian() {
+    require(guardian != address(0), "Guardian is not set");
```

Comment:
Is this check necessary? `msg.sender` will never be 0.

Reply:
Unless I am missing something, this is checking that guardian variable has been set to an address (as opposed to checking that `msg.sender` is never 0).

Reply:
Ah, I see what you are saying in the line below.

Reply:
And probably fine to update the comment to just saying "can only be called when the caller is the guardian" or similar.